### ISSUE TITLE
Why is js() a function instead of a variable?

### ISSUE THREAD

**Issue by marcoos** [tantek/cassis#5]
Shouldn't this:

``` php
function js() {
 return ("00"==false);
}
```

be replaced with this:

``` php
$js = ("00" == false);
```

This way, there would be less function calls and the "00"==false expression would only be evaluated once per script run.

**Comment by aaronpk** [comment]
I don't think this will work as a variable, since in PHP, global variables must be declared at the beginning of the function. For example, in order for a function to be able to see the `$in_js` variable, it would need to look like this:

```
function example() {
  global $in_js;
  if($in_js) {
    // javascript
  } else {
    // php
  }
}
```

The advantage of using the `js()` function is that functions are available in the global scope.

**Comment by tantek** [comment]
Just switched to returning true or false depending on which comment-code path so that should be even faster (for any modern js jit / or PHP processors).
